### PR TITLE
#131 Create endpoints for Users

### PR DESCRIPTION
**Pull request by laila1222**
**Created endpoints to:**

- get user by id

- create new user

- delete user

**Wrote documentation in swagger**
**Commented out modules router, which gives an error**

### CODE REVIEW

File: src/server/api/controllers/users.controller.js [hunk 1]
```diff
@@ -1,10 +1,36 @@
...
+// Create new user
+const createUser = (body) => {
+  return knex('users').insert({
+    name: body.name,
+    fk_role_id: body.fk_role_id,
+  });
+};
+
+// Delete user by id
+const deleteUser = (user_id) => {
```
Comment (OlkaA) [inline src/server/api/controllers/users.controller.js]: Small note: I would change the name of this function to deleteUserById instead deleteUser

**Review by OlkaA — APPROVED** [review]

**Review by OlkaA — COMMENTED** [review]
Everything works perfectly. ✔️ 👍

File: src/server/api/controllers/users.controller.js (hunk 1 above)
Comment (laila1222) [reply src/server/api/controllers/users.controller.js]: Thanks, changed it!

File: src/server/api/controllers/users.controller.js [hunk 2]
```diff
@@ -1,10 +1,36 @@
...
+    .select('*');
+};
+
+// Create new user
+const createUser = (body) => {
+  return knex('users').insert({
+    name: body.name,
+    fk_role_id: body.fk_role_id,
+  });
+};
```
Comment (bytewiz) [inline src/server/api/controllers/users.controller.js]: Could be shortened if you want: 
```javascript
const createUser = ({ name, fk_role_id }) => {
  return knex('users').insert({
    name,
    fk_role_id,
  });
};
```

File: src/server/api/controllers/users.controller.js [hunk 3]
```diff
@@ -1,10 +1,36 @@
...
+
+// Delete user by id
+const deleteUserById = (user_id) => {
+  return knex('users')
+    .where({ user_id: user_id })
```
Comment (bytewiz) [inline src/server/api/controllers/users.controller.js]: could be shortened: `.where({ user_id })`

**Review by bytewiz — APPROVED** [review]
@Laila1222 Adde some comments you can add in if you want to 👍 🔥

**Pull request merged**